### PR TITLE
Paulg13's hometown file :smile:

### LINKED ISSUE #16: paulg13's checklist
- [x] Create an issue
- [x] Create a branch
- [ ] Create a Pull Request
- [ ] Collaborate!
- [ ] Merge

### PR DESCRIPTION
**Pull request by paulg13**
Just adding a hometown file for Des. Plaines,, IL with my recommendations.

Closes #16

### CODE REVIEW

File: paulg13-hometown.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Garibaldis
+## Great Places to Great
```
Comment (cynthiaj13) [inline paulg13-hometown.md]: Wait...great places to great? To eat?

File: paulg13-hometown.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Garibaldis
```
Comment (cynthiaj13) [inline paulg13-hometown.md:1]: What kind of food is here?

**Review by yash1426 — CHANGES_REQUESTED** [review]
What is the best :pizza:  place in Des Plaines

**Review by yash1426 — APPROVED** [review]

File: paulg13-hometown.md (hunk 2 above)
Comment (paulg13) [reply paulg13-hometown.md:1]: PIZZA...ITALIAN

Comment (cynthiaj13) [reply paulg13-hometown.md:1]: good italian beef??

Comment (paulg13) [reply paulg13-hometown.md:1]: Never tried their beef...but bet its good

**Pull request merged**